### DISCUSSION TITLE
Please i need help i don't understand

### DISCUSSION THREAD

**Discussion by Demavrb** [ApolloAuto/apollo#15575] [Q&A]
Hello I want to do a thesis for my undergraduate degree and run the simulation to study it but I don't understand what can help me from all these files and where to start. If anyone is willing to explain a little bit what I am doing I would appreciate it very much.Thanks!

**Comment by Reebs637** [comment]
Hey I can help you with your thesis contact me on WhatsApp at +44 7460 560474 Thanks!
